src/compare_covermaps.py

```diff
@@ -76,6 +76,30 @@
     "Zambia2019": DATA_PATH + "Zambia_CEO_2019.csv",
 }
 
+# Hold year of test files
+TEST_YEARS = {
+    "Kenya": 2019,
+    "Togo": 2019,
+    "Tanzania": 2019,
+    "Malawi": 2020,
+    "Mali": 2019,
+    "Namibia": 2020,
+    "Rwanda": 2019,
+    "Uganda": 2019,
+    "Zambia": 2019,
+    "Senegal": 2022,
+    "Hawaii": 2020,
+    "BlueNile2020": 2020,
+    "BlueNile2019": 2019,
+    "AlGadaref2019": 2019,
+    "GedarefDarfurAlJazirah2022": 2022,
+    "BureJimma2019": 2019,
+    "BureJimma2020": 2020,
+    "Tigray2021": 2021,
+    "Tigray2020": 2020,
+    "Zambia2019": 2019,
+}
```

Comment:
I don't really love that every time an intercomparison is made this file needs to be updated. I think it would be cleaner to minimize (or eliminate) changes to this file, and set the configuration (dataset name, admin code, year) in each respective notebook. In my view it'll be a bit more modular and pleasant to use that way.

Of course that script that generates all the intercomparisons would need to be updated or deleted, but since this is the primary way we are doing intercomparisons, I would be alright with that.

What do you think? @hannah-rae